modules/navier_stokes/src/userobjects/INSFVRhieChowInterpolator.C

```diff
@@ -202,6 +206,11 @@ INSFVRhieChowInterpolator::INSFVRhieChowInterpolator(const InputParameters & par
 
   if (_velocity_interp_method != Moose::FV::InterpMethod::Average)
     fillARead();
+
+  if (!getExecuteOnEnum().contains(EXEC_PRE_KERNELS) || getExecuteOnEnum().size() > 2 ||
```

Comment:
I feel like we should error if it doesn't contain `EXEC_PRE_KERNELS` ... but I suppose if you're doing passive scalar transport (or similar) in an app different from the Navier-Stokes solve, and so are reading the A coefficients from aux variables, then you don't actually need `EXEC_PRE_KERNELS` do you?

Reply:
well the execution is very cheap since we wont have any residual objects if we do passive scalar transport. 

I think you are right, I should just make EXEC_INITIAL default and we ll take the small hit on performance to avoid the trouble.